### PR TITLE
Cold front warning (Feature Freeze on March 21st)

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Feature freeze starting March 21st, our issue count is out of control. End date/conditions are to be determined.

PRs opened prior to the freeze are exempt, as are sprite changes.

Debating whether to allow refactors this time since they often end up breaking even more things.

### CODE REVIEW

**Comment by adamkad1** [comment]
>405 -'es

**Comment by ma44** [comment]
merge when

**Comment by KorPhaeron** [comment]
@tgstation/commit-access

**Comment by Armhulen** [comment]
Make planetstation changes legal!

**Comment by Qustinnus** [comment]
kor pls dont do this to us

**Comment by Cyberboss** [comment]
Denied, fix the broken ass game

**Comment by AlexTheSysop42** [comment]
#34865

**Comment by arsserpentarium** [comment]
Fuck.Can I make PR before march 21?

**Comment by AlexTheSysop42** [comment]
yes

**Comment by Iamgoofball** [comment]
shiningending1-1050x656

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,407 +1,3 @@
 # CONTRIBUTING
 
-## Reporting Issues
```
Comment (kevinz000) [inline .github/CONTRIBUTING.md]: no one will even notice the removal

**Comment by Supermichael777** [comment]
when are we implementing 3/4ths humans?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Shadowflame909) [reply .github/CONTRIBUTING.md]: REE - yeah thats true. no one uses it.

**Comment by Thunder12345** [comment]
image

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,407 +1,3 @@
...
-* Code where one line of code is split across mutiple lines (except for multiple, separate strings and comments; in those cases, existing longer lines must not be split up)
-* Code adding, removing, or updating the availability of alien races/species/human mutants without prior approval. Pull requests attempting to add or remove features from said races/species/mutants require prior approval as well.
-
-Just because something isn't on this list doesn't mean that it's acceptable. Use common sense above all else.
-
-## A word on Git
-Yes, we know that the files have a tonne of mixed Windows and Linux line endings. Attempts to fix this have been met with less than stellar success, and as such we have decided to give up caring until there comes a time when it matters.
-
-Therefore, EOF settings of main repo are forbidden territory one must avoid wandering into, at risk of losing body and/or mind to the Git gods.
+We will only be accepting catgirl related content going forward. Thank you for your understanding.
```
Comment (Bawhoppen) [inline .github/CONTRIBUTING.md]: Nobody noticed what this was actually *adding*...

**Review by ghost — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (ma44) [reply .github/CONTRIBUTING.md]: I did :(

**Comment by ShizCalev** [comment]
Add maps too so I can bash my head against the wall and switch the stations over to multilayer piping imo

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (davethwave) [reply .github/CONTRIBUTING.md]: Well no one will actually follow it and we all know it's a joke one way or another.

**Comment by davethwave** [comment]
So while this freeze is happening only fix PR's and such things will be allowed through?

**Comment by Cyberboss** [comment]
Yes

**Comment by pubby** [comment]
what about april fools

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (Shadowflame909) [reply .github/CONTRIBUTING.md]: so your telling me a cat girl removal PR is in order?

**Comment by Shadowflame909** [comment]
By fix Pr's does that include removals and nerfs? Or just issue fixes? Hopefully the latter.

**Comment by davethwave** [comment]
Though I'm guessing but removal of certain code could create issues while nerfs/buffs would probably be closed due as that would mean that exceptions and that would draw attention from the work that needs to be done which is trying to fix the 1k of issues that are piling up.

**Comment by Cyberboss** [comment]
We aren't repeating the removals as balance memes from last time

**Comment by praisenarsie** [comment]
> what about april fools

**Comment by GunHog** [comment]
We have a branch for April Fool's day, I think. Your PRs can just go to that!

**Comment by KorPhaeron** [comment]
April fools posting goes here https://github.com/tgstation/tgstation/issues/36418

**Comment by banthebantz** [comment]
>issue count falls just under 1000
>yeah it's fine guys feature freeze over

**Comment by 81Denton** [comment]
Does this also affect map tweaks?

**Comment by MrStonedOne** [comment]
>Debating whether to allow refactors this time since they often end up breaking even more things.

Just a note @KorPhaeron that decision will likely decide rather or not tgui2.0 has to wait for the freeze to end.

**Comment by Cyberboss** [comment]
I think we can provide an exemption for that

**Pull request closed without merging**